recipe/meta.yaml

```diff
@@ -1,20 +1,20 @@
-{% set version = "1.11.16" %}
+{% set version = "1.11.20" %}
 
 package:
   name: django
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/D/Django/Django-{{ version }}.tar.gz
-  sha256: 29268cc47816a44f27308e60f71da635f549c47d8a1d003b28de55141df75791
+  sha256: 43a99da08fee329480d27860d68279945b7d8bf7b537388ee2c8938c709b2041
   patches:
     # Hard-code gdal and geos' paths to the corresponding conda packages.
     - libgdal.patch
     - libgeos.patch
 
 build:
   number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  script: python -m pip install --no-deps --ignore-installed .
```

Comment:
Please see https://conda-forge.org/docs/maintainer/adding_pkgs.html#use-pip for the right line incantation we should be using here.